### ISSUE TITLE
Are you still interested in the Quest?

### ISSUE THREAD

**Issue by teccheck** [Migushthe2nd/QuestFirmwareGrabber#2]
Hi there, I've recently come across this project and it peaked my interest. I've been reverse engineering the Quest for quite a while and wanted to ask if you are still into it

EDIT: Btw. I was able to download a consumer firmware zip (Quest 1 v50)

**Comment by Migushthe2nd** [comment]
Yeah I'd be interested, depending on what you're working on. Could you contact me on discord? `@migush`

**Comment by ilovecats4606** [comment]
Hey @Migushthe2nd @teccheck if you guys are still intrested, I've made a discord server for rooting Q123. We already got partial escalate access with Zygote injection, [https://discord.gg/ABCXxDyqrH](https://discord.gg/ABCXxDyqrH)
